app/src/org/commcare/views/widgets/GridWidget.java

```diff
@@ -78,7 +79,8 @@ public GridWidget(Context context, FormEntryPrompt prompt,
         // The max width of an icon in a given column. Used to line
```

Comment:
the comment is just referring to the width though (the fact that it's used to line up and auto-fit the columns)